tests/test_util.go

```diff
@@ -297,3 +323,72 @@ func removeSecret(cli kubernetes.Interface, secretName, ns string) error {
 	}
 	return nil
 }
+
+func waitUntilArangoDeploymentHealthy(deployment *api.ArangoDeployment, DBClient driver.Client, k8sClient kubernetes.Interface, versionString ...string) error {
```

Comment:
Add comment describing what the function does.

Reply:
Replace `versionString ...string` with `requiredVersion driver.Version`.
If that is a non-empty string, create the predicate.

Reason: I think we may want to add other checks in the future and you can only have 1 optional argument.